commands/translation_stat/tr_core.py

```diff
@@ -0,0 +1,377 @@
+from github import Github, Repository, GithubObject
+import json
+import os
+from threading import Lock
+from contextlib import nullcontext
+import re
+from datetime import datetime
+
+g: Github = None
+repo: Repository = None
+
+# repo = g.get_repo("placeTW/website")
+main_lang = "en"
+json_matches: dict[str, list[str]] = {
+    "art-pieces.json": [r".*#(title|blurb|desc)"],
+    "translation.json": [r".*"],
+}
+
+lang_require_update: bool = False
+
+def float_to_percentage(target: float) -> int:
+    return int(target * 100)
+
+def check_file_include_rule(path: str) -> bool:
+    return re.search(
+        r"public\/locales\/.*\/(art-pieces\.json|translation\.json)", path
```

Comment:
The filenames shouldn't be hard coded.
I noticed you have the `json_matches` dict, you should get the filenames from the keys of that dict

Reply:
I wanted an full path match because I'm concerned about having non-wanted files with the same filename but in another path (subfolders)

Reply:
You can do something like `"public\/locales\/.*\/({'|'.join(list(json_matches.keys()))})"` (not sure if this would work exactly but something like this)

Reply:
I see